dask/dataframe/core.py

```diff
@@ -5086,6 +5086,26 @@ def applymap(self, func, meta="__no_default__"):
     def round(self, decimals=0):
         return elemwise(M.round, self, decimals)
 
+    @derived_from(pd.DataFrame)
+    def nunique(self, split_every=False):
+        nunique_list = []
+        for col in self.columns:
+            nunique = Series.nunique(self[col], split_every=split_every)
+            nunique.name = self[col].name
+            nunique_list.append(nunique)
+        name = "series-" + tokenize(*nunique_list)
+        layer = {
+            (name, 0): (
+                apply,
+                pd.Series,
+                [[(s._name, 0) for s in nunique_list]],
+                {"index": [s.name for s in nunique_list]},
+            )
+        }
+        graph = HighLevelGraph.from_collections(name, layer, dependencies=nunique_list)
+        meta = pd.Series(index=[scalar.name for scalar in nunique_list], dtype=int)
```

Comment:
ahh ok! I was wondering how to do this

Reply:
Hmm normally meta doesn't have any rows. So you really just want an empty series with `dtype=int`. But since we don't want to only support panges, probably Ian is right that the simplest way is to emulate the function on the `meta`. You can use the regular empty one though:

```suggestion
        meta = self._meta.nunique()
```